ekf_estimator/src/ekf_estimator.cpp

```diff
@@ -1,22 +1,42 @@
 #include "ekf_estimator/ekf_estimator.h"
 
+
+const int numofStates = 6;
+const int num_feet_ = 4;
```

Comment:
You can make these static const ints that are member variables of the class so they aren't in the global scope.